src/public/style.css

```diff
@@ -2,13 +2,14 @@ body {
   background-color: #d9e6fc;
 }
 h1 {
+  display: inline;
```

Comment:
This is not necessary, you just need to get the `<header>` inside the `<body>` tags.